components/switch/switch.js

```diff
@@ -11,11 +11,25 @@ import { SwitchMixin } from './switch-mixin.js';
  */
 class Switch extends SwitchMixin(LitElement) {
 
+	static get styles() {
+		return [super.styles, css`
+			.d2l-switch-icon-off svg {
+				color: var(--d2l-color-ferrite);
```

Comment:
I think if you set this on the `d2l-icon-custom` (`.d2l-switch-off > d2l-icon-custom`) then the media query isn't necessary?